src/commands/CreateCommand.js

```diff
@@ -0,0 +1,33 @@
+import Command from "../Command";
+import ConfigUtilities from "../ConfigUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import NpmUtilities from "../NpmUtilities";
+import path from "path";
+
+export default class LsCommand extends Command {
```

Comment:
In my repo, I use Yeoman to do package generation, which has been working great. Perhaps this command isn't really necessary?